docs/sources/setup/install/helm/install-scalable/_index.md

```diff
@@ -91,5 +134,77 @@ It is not recommended to run scalable mode with `filesystem` storage.
        helm upgrade --values values.yaml loki grafana/loki
        ```
 
+## AWS S3 Configuration
+
+To configure Loki to use AWS S3 as the object storage, you need to provide the following values in the `values.yaml` file:
```

Comment:
Why not combine this with the example above?  Curious as to why it's called out separately.

Reply:
So I was going to ask you about this one. I currently have the first example using MinIO for test purposes and then provide the cloud storage configurations after. I have added the S3 one for now but currently testing Azure blog to also add next